### PR TITLE
preview REST APIs of azure-mysqldb and azure-postgresqldb will be retired on January 31, 2019

### PR DESCRIPTION
**Pull request by bingosummer**

### CODE REVIEW

**Comment by bingosummer** [comment]
CC @zhongyi-zhang

File: docs-content/release-notes.html.md.erb [hunk 1]
```diff
@@ -22,6 +22,8 @@ owner: Partners
 
 **Release Date:** August 30, 2018
 
+Note: If you are using `azure-mysqldb` and `azure-postgresqldb`, you need to upgrade the tile to `v1.10.0+` before **January 31, 2019**. Because the REST API version `2017-04-30-preview` of `azure-mysqldb` and `azure-postgresqldb` will be retired on January 31, 2019.
```
Comment (zhongyi-zhang) [inline docs-content/release-notes.html.md.erb]: Is "azure-mysqldb **or** azure-postgresqldb" better?

**Comment by zhongyi-zhang** [comment]
LGTM.

**Comment by jknostman3** [comment]
@bingosummer Thanks!

**Pull request merged**